oap-server/server-alarm-plugin/src/test/java/org/apache/skywalking/oap/server/core/alarm/provider/expression/ExpressionTest.java

```diff
@@ -70,7 +70,7 @@ public void testEvalWithEmptyContext() {
 
     @Test
     public void testCompile() {
-        String expr = " a && b ";
+        String expr = " a && * b ";
```

Comment:
As logs said

> Error:  Errors: 
Error:    ExpressionTest.testCompile:75 � Compile [Error: [Error: was expecting type: ja...
[INFO] 
Error:  Tests run: 50, Failures: 0, Errors: 1, Skipped: 0

Seems this would cause throwing exception. Please recheck.

Reply:
Sorry, fixed the test case